bot/exts/utilities/bookmark.py

```diff
@@ -87,8 +86,8 @@ async def send_reaction_embed(
         await message.add_reaction(BOOKMARK_EMOJI)
         return message
 
-    @whitelist_override(channels=WHITELISTED_CHANNELS, categories=WHITELISTED_CATEGORIES)
     @commands.command(name="bookmark", aliases=("bm", "pin"))
+    @whitelist_override(roles=(Roles.everyone,))
```

Comment:
This should go above commands.command to be consistent with how it was before; also keeps git diff to a minimum.

Reply:
It was above, and that's why it didn't work.

Reply:
This actually needs to go below the command decorator, otherwise it has no effect. I believe it did not work previously.

Reply:
(yes, no one tested the old PR besides mina)

Reply:
Ah, okay.